modules/installation-ibm-z-supported-operators.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_z/installing-ibm-z.adoc
+
+[id="installation-ibm-z-supported-operators_{context}"]
+= Supported Operators on IBM Z
+
+As a cluster administrator, you can install an Operator from the OperatorHub using the OpenShift Container Platform web console or the CLI. You can then subscribe the Operator to one or more namespaces to make it available for developers on your cluster. Following table lists all Operators that are known to be supported for {product-title} on IBM Z.
+
+[cols="8,8",options="header"]
+|===
+
+|Web UI Operator name
```

Comment:
s/Web UI/Web console

Reply:
Fixed in 8515b01b7c7e293649df73735b14247f0a95626c